cmd/pfaFindFast/efficient.go

```diff
@@ -0,0 +1,337 @@
+package main
```

Comment:
Comparing this file with main cmd/faFindFast/efficient.go on a tool like https://www.diffchecker.com/text-compare/ shows that the only changes are dna vs pdna